_data/tichnas/tichnas.md

```diff
@@ -0,0 +1,7 @@
+---
+name: tichnas
+institution: IIIT-H
+profile_pic: S.jpg
+quote: I don't always graduage, but when I do, I barely graduate ;)
```

Comment:
Perhaps a typo in your quote, change _graduage_ to _graduate_?

Reply:
I'm really sorry about that. I have corrected it now.